### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by EridianBrian** [needim/noty#311]
I just wanted to leave a note here, I love your script but the documentation is extremely lacking for beginners... I ended up having to do a lot of work to get noty to work in what I needed to do (examples bound to press buttons are one thing, but notifying on form errors, etc is quite another).

I needed to display CodeIgniter flash data after a page reload for login failure, you had no working examples of how to get JavaScript / jQuery to display the notification and I was left entirely to my own devices.

Please put some more detailed documentation up to help people, I ended up getting where I needed to in the end but took 10x longer than it really should have...

**Comment by stamminator** [comment]
Agreed. Same issues with me. Been using noty for several months now and I still don't know how to do some basic things

**Comment by siddharthbhagwan** [comment]
The documentation contains an 'Installation' section and a 'creating a noty' section that pretty much mention everything you need to get started. @EridianBrian could you mention a couple of things that you thought were not mentioned so the documentation can be updated accordingly? @stamminator Could you give examples of some basic things are as well?

**Comment by EridianBrian** [comment]
Hi @stamminator 
My issue is explained on stackoverflow here: http://stackoverflow.com/questions/34824703/using-jquery-noty-within-codeigniter-view-for-flashdata-page-reload/34825421#34825421

If you require additional information, please let me know but my problem and solution is listed there...

**Comment by needim** [comment]
related #354
